internal/utils/utils.go

```diff
@@ -0,0 +1,33 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	semver "github.com/blang/semver/v4"
+)
+
+func GetNextOCPMinorVersion(versionString string) (*semver.Version, error) {
+	v, err := ToSemver(versionString)
```

Comment:
Parsing the OCP version should use strict semver parsing, not our tolerant "maxOpenshiftVersion" parsing.